src/shared_realm.hpp

```diff
@@ -210,6 +212,7 @@ class Realm : public std::enable_shared_from_this<Realm> {
     void invalidate();
     bool compact();
     void write_copy(StringData path, BinaryData encryption_key);
+    OwnedBinaryData write_copy_to_mem();
```

Comment:
Abbreviating "memory" is unnecessary.

"Writing" feels like weird terminology to use when talking about an in-memory operation. I don't have any better alternatives in mind, though.